plugins/org.eclipse.jst.j2ee/archiveops/org/eclipse/jst/j2ee/internal/archive/JavaEEArchiveUtilities.java

```diff
@@ -918,6 +918,14 @@ private static void wrapArchive(final IArchive simpleArchive, final IPath deploy
 	private static final char[] MESSAGEDRIVEN = "Ljavax/ejb/MessageDriven;".toCharArray();//$NON-NLS-1$
 
 	private static final char[] SINGLETON = "Ljavax/ejb/Singleton;".toCharArray();//$NON-NLS-1$
+	
+	private static final char[] STATELESS_JAKARTA = "Ljakarta/ejb/Stateless;".toCharArray();//$NON-NLS-1$
+
+	private static final char[] STATEFUL_JAKARTA = "Ljakarta/ejb/Stateful;".toCharArray();//$NON-NLS-1$
+
+	private static final char[] MESSAGEDRIVEN_JAKARTA = "Ljakarta/ejb/MessageDriven;".toCharArray();//$NON-NLS-1$
+
+	private static final char[] SINGLETON_JAKARTA = "Ljakarta/ejb/Singleton;".toCharArray();//$NON-NLS-1$
```

Comment:
I think it would be better to rename the old constants to JAVA_EE_* (e.g. `SINGLETON_JAVA_EE`) and just use the old names for the new constants, i.e. `STATELESS` instead of `STATELESS_JAKARTA`.

Reply:
Renaming is done.